spec/requests/categories_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Categories', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
```

Comment:
- [ ] Kindly add tests for your route, it's not good practice to have test files without files.